library/Fission/Mail/SendInBlue/Types.hs

```diff
@@ -0,0 +1,53 @@
+module Fission.Mail.SendInBlue.Types 
+  ( SendEmailReq(..)
+  , CustomParams(..)
+  , SendEmailRes(..)
+  , module Fission.Mail.SendInBlue.ApiKey.Types
+  , module Fission.Mail.SendInBlue.TemplateId.Types
+  ) where
+
+import           Fission.Prelude
+import           Fission.Mail.Recipient.Types
+import           Fission.Mail.SendInBlue.ApiKey.Types
+import           Fission.Mail.SendInBlue.TemplateId.Types
+
+import           Fission.User.Username.Types
+import           Servant.Client
+
+import qualified Data.Vector as Vector
+
+
+data SendEmailReq = SendEmailReq
+  { templateId :: !TemplateId
+  , to :: ![Recipient]
+  , params :: !CustomParams
```

Comment:
What is `CustomParams`? Template options?

Reply:
yup exactly

I originally had it `Params` but that was so vague that I changed it to `CustomParams`, but `TemplateOptions` is a more descriptive type name